README.md

```diff
@@ -27,6 +27,20 @@ documentation at https://articlequality.readthedocs.io .
                      'ga': 0.21617801281707663,
                      'start': 0.13906813268582238}}
 
+## Install
+
+### Requirements
+
+* Python 3.5, 3.6 or 3.7
+* All the system requirements of [revscoring](https://github.com/wikimedia/revscoring)
+
+### Installation steps
+
+1. clone this repository
+2. install its dependencies `python -m pip -r requirements.txt`
```

Comment:
Shouldn't this command just be `pip install -r requirements.txt`

Reply:
`python -m pip` is the explicitly recommended way:
* [official docs](https://docs.python.org/3/installing/index.html#basic-usage)
* [more reasoning by a python core dev](https://snarky.ca/why-you-should-use-python-m-pip/)

Reply:
That's so strange, but it's fair.  The author's argument that "you don't run pip all of the time" is very wrong in my experience, but I think the point still stands.